config/core/db-configs.yaml

```diff
@@ -1,5 +1,9 @@
 db_configs:
 
+  api:
+    db_type: kernelci_api
+    url: http://localhost:8001/
+
```

Comment:
I think it's a bit too early to add this, which is why it hasn't been added yet. The pipeline has its own YAML config to define it instead.

Reply:
But it needs to be added to run the command:
```
./kci_data create_user --username admin --password admin --db-config api --is-admin
```

Reply:
Yes but that's already defined in `kernelci-pipeline`, so it's a grey area. My point is that support for KernelCI API is not actually ready yet in `kernelci-core`. Calling it `api` is also not descriptive enough, so I think it will lead to issues later when we have an actual production one online. You can provide `--yaml-config` to specify where the YAML configuration is when running `kci_data`, and use the YAML file from `kernelci-pipaline` which has an entry for the local API.

Reply:
Okay. But then it will be dependent on `pipeline`. Why would we do that when we can directly add configurations in `core`? And what about when we just need `core` utilities without setting up the `pipeline`?

Reply:
The code in `kernelci-core` is not ready for communicating with the API, we only have a few things in place that only work with the code in `kernelci-pipeline` and it's likely to have to be rewritten as the API is evolving and support for `kernelci-backend` will be deprecated and removed. So yes I think we might ultimately have entries in `api-configs.yaml` or something for the new API server but right now I don't feel comfortable with adding it here. It seems too early, some people might try to use it and hit some issues. So it's a grey area, and because we don't really need it I think it's safer to leave it out for now. If in practice it's causing too many issues then I guess we can have it merged.

Reply:
Okay. Let's leave it out as of now and I'll drop the commit.